SwiftLeeds/Views/About/ContributorsPage.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ContributorsPage.swift
+//  SwiftLeeds
+//
+//  Created by LUCKY AGARWAL on 20/07/22.
+//
+
+import SwiftUI
+
+struct ContributorsPage: View {
+	var body: some View {
+		SwiftLeedsContainer {
+			ScrollView {
+				content
+			}
+		}
+		.edgesIgnoringSafeArea(.top)
+	}
+	
+	private var content: some View {
+		VStack(spacing: Padding.cellGap) {
+			HeaderView(
+				title: "Our Team",
+				imageAssetName: Assets.Image.swiftLeedsIcon,
+				backgroundImageAssetName: Assets.Image.playhouseImage
+			)
+			VStack(spacing: Padding.cellGap) {
```

Comment:
As there's only one view inside the VStack, not sure what the Spacing is for?

Reply:
yeah. that's unnecessary.